tests/console-backend-service/internal/configurer/testbundle.go

```diff
@@ -0,0 +1,160 @@
+package configurer
+
+import (
+	"github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+	tester "github.com/kyma-project/kyma/tests/console-backend-service"
+	"github.com/kyma-project/kyma/tests/console-backend-service/internal/waiter"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1Type "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type TestBundleConfig struct {
+	ConfigMap TestBundleConfigMap
+
+	ClusterServiceBrokerName        string   `envconfig:"default=helm-broker"`
+	ClusterServiceClassExternalName string   `envconfig:"default=testing"`
+	ClusterServicePlanExternalNames []string `envconfig:"default=full,minimal"`
+}
+
+type TestBundleConfigMap struct {
+	Name          string `envconfig:"default=helm-broker"`
```

Comment:
Same question as for service broker. What do you think about adding information that it is for CBS tests?